redisearch/aggregation.py

```diff
@@ -108,6 +108,49 @@ def build_args(self):
                 ret += ['AS', reducer._alias]
         return ret
 
+class Projection(object):
+    """
+    This object automatically created in the `AggregateRequest.apply()`
+    """
+
+    def __init__(self, alias, projector):
+
+        self.alias = alias
+        self.projector = projector
+
+    def build_args(self):
+        ret = ['APPLY', self.projector]
+        if self.alias:
+            ret += ['AS', self.alias]
+
+        return ret
+
+class SortBy(object):
+    """
+    This object automatically created in the `AggregateRequest.sort_by()`
+    """
+
+    def __init__(self, fields, max):
```

Comment:
Maybe we can set max to be 0 by default, then it will not be mandatory, what do you think?